### PR TITLE
Show vehicle page

### PR DESCRIPTION
**Pull request by gdhaison**
===> Title: Show vehicle page

===> Redmine ticket: https://edu-redmine.sun-asterisk.vn/issues/19139 

===> Attachment files: 
Screenshot from 2019-11-06 23-38-50

### CODE REVIEW

**Comment by gdhaison** [comment]
ready

File: app/controllers/vehicles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class VehiclesController < ApplicationController
+  def show
+    @vehicle = Vehicle.find_by id: params[:id]
```
Comment (ngocvt-0484) [inline app/controllers/vehicles_controller.rb]: trong trường hợp @vehicle k tìm thấy thì sao e?

Comment (gdhaison) [reply app/controllers/vehicles_controller.rb]: để em sửa ạ

**Comment by gdhaison** [comment]
ready

**Pull request merged**